tests/unit/style/media_queries.rs

```diff
@@ -18,28 +18,35 @@ use style_traits::ToCss;
 pub struct CSSErrorReporterTest;
 
 impl ParseErrorReporter for CSSErrorReporterTest {
-     fn report_error(&self, _input: &mut Parser, _position: SourcePosition, _message: &str) {
-     }
+    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str,
+        servo_url: Option<&ServoUrl>) {
+            let css_url = servo_url.unwrap().clone();
```

Comment:
Lets keep the url as a param, but leave the body of this method empty.
You'll want to rename the parameter to `_url` or `_servo_url` as the `_` will get rid of the unused variable warning